js/ui/notify.js

```diff
@@ -29,4 +39,100 @@ const notify = function(message, /* optional */ type, displayTime) {
     new Toast($notify, options).show();
 };
 
+const createStackContainer = (id) => {
+    const newContainer = $('<div>').appendTo($(domAdapter.getBody()));
+    newContainer.attr('id', id);
+
+    return newContainer;
+};
+
+const getStackContainer = (id) => {
+    if($(`#${id}`).length) {
+        return $(`#${id}`);
+    } else {
+        return createStackContainer(id);
+    }
+};
+
+
+const setContainerClasses = (container, direction) => {
+    container.removeClass();
+    container.addClass('dx-toast-container').addClass(`dx-toast-container-${direction}-direction`);
+};
+
+const setContainerStyles = (container, direction, position) => {
+    const styles = typeof position === 'string'
+        ? getPositionStylesByAlias(container, direction, position)
+        : getPositionStylesByCoordinates(container, direction, position);
+
+    container.css(styles);
+};
+
+const getPositionStylesByAlias = (container, direction, position) => {
```

Comment:
let's implement `getCoordinatesByAlias` function instead of this one to get rid of copypaste